src/app/shapes/Rectangle.ts

```diff
@@ -0,0 +1,35 @@
+  
+class Rectangle extends Shape {
+  private width: number;
+  private height: number;
+
+  private strokeWidth: number = 2;
```

Comment:
This will have to be changed. I think Micah will be refactoring this in all the classes!